packages/foreman/rubygem-rkerberos/rubygem-rkerberos.spec

```diff
@@ -0,0 +1,98 @@
+# template: scl
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name rkerberos
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.1.5
+Release: 2%{?dist}
```

Comment:
Is the 2 intentional? EPEL7 has 0.1.3-6, so 0.1.5-1 would be newer anyways.
And if we want to be newer than Fedora, it needs to be >15 (see https://src.fedoraproject.org/rpms/rubygem-rkerberos/blob/master/f/rubygem-rkerberos.spec)